README.md

```diff
@@ -176,7 +176,8 @@ protobuf {
   // Locate the codegen plugins
   plugins {
     // Locate a plugin with name 'grpc'. This step is optional.
-    // If you don't locate it, protoc will try to use "protoc-gen-grpc" from
+    // If you leave it empty, it uses the current directory.
+    // If you don't specify any plugins, protoc will try to use "protoc-gen-grpc" from
```

Comment:
"any plugins" isn't fully accurate, because you can define one plugin and not another. I think just saying "it" may be clearest, to align with the previous sentence.